File: Python-Assignment/Chapter-Three/Question17.java
public class Question17{
   public static void main(String []args){
 
	for (int outerLoop = 1; outerLoop <= 10; outerLoop++){
		for(int innerLoop = 1; innerLoop <= outerLoop; innerLoop++){
			System.out.print("*");
		}
		System.out.println();
	}

System.out.println();

	for (int outerLoop = 0; outerLoop < 10; outerLoop++){
		for(int innerLoop = 0; innerLoop < (10-outerLoop);innerLoop++){
			System.out.print("*");
}
		System.out.println();
	}
System.out.println();

	for(int outerLoop = 0; outerLoop < 10; outerLoop++){
		for(int spaceLoop = 0; spaceLoop < outerLoop; spaceLoop++){
			System.out.print(" ");
		}
		for(int innerLoop = 0; innerLoop < (10-outerLoop); innerLoop++){
			System.out.print("*");
		}
		System.out.println();
	}
System.out.println();

	for(int outerLoop = 0; outerLoop < 10; outerLoop++){
		for(int spaceLoop = 0; spaceLoop < (10-outerLoop); spaceLoop++){
			System.out.print(" ");
		}
		for(int innerLoop = 0; innerLoop < (outerLoop+1); innerLoop++){
			System.out.print("*");
		}
		System.out.println();
	}
   }

}

File: Python-Assignment/chapterThree/question17.py
for outer_loop in range(0, 10):
	for inner_loop in range(0, outer_loop+1):
		print('*', end='')
	print()

print()

for outer_loop in range(0, 10):
	for inner_loop in range(0, 10-outer_loop):
		print('*', end='')
	print()

print()

for outer_loop in range(10):
	for space_loop in range(0, outer_loop):
		print(' ', end='')
	for inner_loop in range(0, 10-outer_loop):
		print('*',end='')
	print()

print()

for outer_loop in range(10):
	for space_loop in range(0, 10-outer_loop):
		print(' ', end='')
	for inner_loop in range(0, outer_loop+1):
		print('*',end='')
	print()

